### DISCUSSION TITLE
How are paths supposed to specified on Windows system?

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#7701] [Help: Coding & Implementations] [answered]
I'm having problem when the config is being read, it cannot find my word2vec vectors.  I keep getting this error 

OSError: [E884] The pipeline could not be initialized because the vectors could not be found at '"\\networkfileshare\dev\Word2Vec\spacy_word2vec\2.0.0.0"'. If your pipeline was already initialized/trained before, call 'resume_training' instead of 'initialize', or initialize only the components that are new.

I specified it like this in the paths section

`vectors =  "\\networkfileshare\dev\Word2Vec\spacy_word2vec\2.0.0.0"`

(using spacy 3.0.5)

**Comment by polm** [comment, marked as answer]
Hm, the documentation might be a little lacking on this but I think it'll work if you put an `r` in front of it, same as Python.

```
vectors = r"\\networkfileshare\dev\Word2Vec\spacy_word2vec\2.0.0.0"
```

**Reply by polm** [reply]
As another note, I haven't confirmed this on a Windows system yet, but I think you can use forward slashes and they'll be automatically converted. That may be easier if your paths don't contain literal forward slashes, which is technically legal on Windows (where it is not a directory marker).

**Reply by svlandeg** [reply]
I just double checked on Windows, and the behaviour is slightly inconsistent:
* Using a path with backward slashes hard-coded in the config does not work (as the original issue described)
```
[paths]
train = "C:\Users\User\dataset\train"
```
* Overwriting the config value from the command line with backward slashes DOES work! This is also the recommended approach.
```
spacy train --paths.train "C:\Users\User\dataset\train"
```
* Rewriting your paths in the config file with forward slashes, as described by @polm, also works:
```
[paths]
train = "C:/Users/User/dataset/train"
```

So hopefully one of those options will be sufficient to fix your use-case, but agreed we should document this better!
